tests_additional/real_exchanges/test_binance.py

```diff
@@ -28,7 +28,7 @@
 
 
 class TestBinanceRealExchangeTester(RealExchangeTester):
-    EXCHANGE_NAME = "binance"
+    EXCHANGE_NAME = "binance"  # use binance for tests as binance is blocked in the us
```

Comment:
EXCHANGE_NAME = "binanceus"  # use binanceus for tests as binance is blocked in the us
?

Reply:
:laughing: